hyperdome/client/hyperdome_client.py

```diff
@@ -193,19 +185,21 @@ def send_message(self):
         self.message_text_field.clear()
 
         if not (self.is_connected or self.uid):
-            return self.handle_error("not in an active chat")
+            return self.handle_error(Exception("not in an active chat"))
```

Comment:
Probably doesn't matter, but it'd be nice to have a more specific exception

Reply:
I used regular exception just because I changed the handle error to take exceptions, but discards any info about the exception since it's only meant to display messages.

I think I want to change this back to just taking an error message. I want to make a bunch of specific hyperdome exceptions, but those exceptions make the most sense to be raised at the api call since that's what should know if a response was bad.

So this should be temporary, new server API version will be more communicative about the nature of errors so that the specific exceptions can be made and this would be caught at the api layer instead of at the GUI.